extensions/stats/plugin.cc

```diff
@@ -196,46 +196,56 @@ const std::vector<MetricFactory>& PluginRootContext::defaultMetrics() {
       // HTTP, HTTP/2, and GRPC metrics
       MetricFactory{
           "requests_total", MetricType::Counter,
-
           [](const ::Wasm::Common::RequestInfo&) -> uint64_t { return 1; },
-          false},
+          false, count_standard_labels},
       MetricFactory{
           "request_duration_milliseconds", MetricType::Histogram,
           [](const ::Wasm::Common::RequestInfo& request_info) -> uint64_t {
             return request_info.duration /* in nanoseconds */ / 1000000;
           },
-          false},
+          false, count_standard_labels},
       MetricFactory{"request_bytes", MetricType::Histogram,
-
                     [](const ::Wasm::Common::RequestInfo& request_info)
                         -> uint64_t { return request_info.request_size; },
-                    false},
+                    false, count_standard_labels},
       MetricFactory{"response_bytes", MetricType::Histogram,
-
                     [](const ::Wasm::Common::RequestInfo& request_info)
                         -> uint64_t { return request_info.response_size; },
-                    false},
+                    false, count_standard_labels},
+      // GRPC metrics.
+      MetricFactory{
+          "request_messages_count", MetricType::Counter,
+          [](const ::Wasm::Common::RequestInfo& request_info) -> uint64_t {
+            return request_info.request_message_count;
+          },
+          false, count_peer_labels},
```

Comment:
Can you add some comments here about what labels are ignored for this metric?

Reply:
Also would connection_security_policy worth to be added?

Reply:
Or we are considering that it is already tracked by request total, thus it is a duplicated here and this should just focus on the message count.

Reply:
Yeah, I only put peer labels, since the other metrics apply to the outer request rather than individual messages within them. I'm not sure if we want more, so I am starting from the minimal set.